### ISSUE TITLE
Add capability to reference existing net port/address list and traffic classes to a AS3 declaration

### ISSUE THREAD

**Issue by ajgerace** [F5Networks/f5-appsvcs-extension#472]
### Is your feature request related to a problem? Please describe.
Currently AS3 does not support the creation or referencing net port/address-lists or traffic classes. If a customer is utilizing any of these features they migrate those configurations into AS3.  

### Describe the solution you'd like
The creation of these features is currently blocked by other issues. I'm asking for the ability to reference previously created objects similar to the way AS3 supports APM API Protection policies.  

### Describe alternatives you've considered
In the past we've utilized Ansible to deploy imperative commands (bigip_command with tmsh commands to build out the net port-list and traffic classes). This requires the configuration to be built completely with Ansible and makes managing the configuration more challenging.  

 `- name: create ftp port list
      bigip_command:
        commands:
          - create net port-list ftp_ftpse_ports ports add { 21 39000-39100 }
          - create net port-list ftpsi_ports ports add { 990 39101-39200 }
          - create ltm traffic-matching-criteria "{{baseName}}_{{siteName}}_ftpse_tmc_obj" { destination-address-inline "{{ vsAddr }}/32" destination-port-list ftp_ftpse_ports protocol tcp source-address-inline 0.0.0.0 }
          - create ltm traffic-matching-criteria "{{baseName}}_{{siteName}}_ftpsi_tmc_obj" { destination-address-inline "{{ vsAddr }}/32" destination-port-list ftpsi_ports protocol tcp source-address-inline 0.0.0.0 }
      delegate_to: localhost
      tags: ftpvs`   

### Additional context
Add any other context, such as the desired tmsh configuration, about the feature request here.
Example virtual server config referencing previously built objects. 

`                "vs_https_443": {
                    "class": "Service_HTTPS",
                    "iRules": [
                        {
                              "bigip": "/Common/api-irule-v4"
                        }
                    ],
                    "pool": "vs_http_pool",
                    **"portList": {
                      "bigip": "/Common/ftp_ftpse_ports"
                    },**
                    "profileHTTP": {
                        "bigip": "/Common/http_profile"
                    },
                    "profileMultiplex": {
                        "bigip": "/Common/oneconnect_profile"
                    },
                    "redirect80": false,
                    "serverTLS": {
                        "bigip": "/Common/http_app_clientssl"
                    },
                    "snat": "auto",
                    **"traffic-matching-criteria": {
                       "bigip": "/Common/ftp_ftpse_tmc_obj"
                    },**
                    "virtualAddresses": [
                        "10.0.10.101"
                    ]

                }

**Comment by dstokesf5** [comment]
This is tracked in our internal product backlog as AUTOTOOL-1573. However, this is currently blocked on multiple TMSH bugs.

**Comment by dstokesf5** [comment]
Closing as duplicate of #269.
